values-schema.yaml

```diff
@@ -207,6 +221,68 @@ definitions:
       tag:
         type: string
     title: Container image
+    type: object
+  job:
+    additionalProperties: false
+    properties:
+      enabled:
+        default: true
+        type: boolean
+      env:
+        $ref: '#/definitions/env'
+      image:
+        $ref: '#/definitions/image'
+      init:
+        properties:
+          image:
+            $ref: '#/definitions/image'
+          script:
+            $ref: '#/definitions/script'
+          shell:
+            pattern: (sh|bash)
+      mounts:
```

Comment:
I am wondering, how a Otomi-console user can use it?

Reply:
> I am wondering, how a Otomi-console user can use it?

As we spoke about in the chat, @j-zimnowoda is talking about the `mounts` key. If I understand correctly, @j-zimnowoda thinks it might be confusing to not expose mounts to the otomi-console but still allow users to define it in values. 

So I guess that implies we don't expose mounts by default. And if it's not by default, then the property also isn't mandatory in the values schema (@j-zimnowoda is this correct?)

What is your input on this @Morriz? I have just defined it here because there's a property in the Job chart that would match also having a definition in the values schema. I didn't think it would have any other implications other than giving the users the option to define mounts for their jobs, I can't imagine it wouldn't be useful.

Reply:
I agree, it is confusing. So far we let them mount configmaps and secrets already, but that might not be enough.

Use case: user wants a config file mounted. How can they? I think we should quickly allow for users to add configmaps as well, so that they can mount those. Please add that to this PR

Reply:
I have defined some behaviour for it under the `volumes` definition in this `values-schema.yaml`: 

```volumes:
    items:
      additionalProperties: false
      properties:
        name:
          description: Name must match mount name.
          type: string
        configMap:
          properties:
            name:
              type: string
```

Is this enough? Right now it's only in line with the `otomi-core` chart.